ministryofjustice/analytics-platform-control-panel-frontend
# Provide front-end for auth0 logging in

## What

[Trello](https://trello.com/c/TdBYSyTp/424-cp-ui-auth0-js-and-fe)

Modularise front-end JS, hook up auth0 lock, show logged in username in header. Temporarily dump JWT out to logged in page. Next PR will do something with this, like create a user on the fly if none exists, or log in if one does.

## How to review

1. Check out branch
2. `npm install`
3. Visit `http://localhost:3000/`
4. Luxuriate in the glow of the Auth0 template
5. Log in using whatever creds you have in the API client (probably github)
6. Observe your JWT stuff spilled out all over the logged in page
7. Check you can log out again with the 'Sign out' link in the header
